src/backend/src/utils/projects.utils.ts

```diff
@@ -117,6 +118,7 @@ export const projectTransformer = (
     bomLink: project.bomLink ?? undefined,
     rules: project.rules,
     duration: calculateDuration(project.workPackages),
+    endDate: calculateProjectEndDate(project.workPackages),
```

Comment:
this line isn't compiling for me because workpackages don't naturally have the endDate field (but they do if you run them through the work package transformer)